shipyard_blueprints/googledrive/shipyard_googledrive/drive_utils.py

```diff
@@ -219,7 +237,7 @@ def get_drive_id(drive_id: str, service) -> Union[str, None]:
 
     """
     try:
-        if len(drive_id) == 33 and str(drive_id).startswith("OAJ"):
+        if len(drive_id) == 19 and str(drive_id).startswith("0A"):
```

Comment:
curious why this condition changed? It would be worth considering making a helper function in the drive utils such as `is_drive_{type}` so that if the criteria needs to be changed again in the future it can be done in 1 place or at the very least add a comment of why char len 19 that begin with 0A is handled differently

Reply:
This was actually incorrect to begin with. The ID for a folder is 33 chars long, while for a drive it is 19. And the prefix for drives is actually just `0A...` not `OA....`